frontend/packages/dev-console/src/components/health-checks/__tests__/create-health-check-probe-data.ts

```diff
@@ -0,0 +1,494 @@
+import { AppResources } from '../../edit-application/edit-application-types';
+import { HealthChecksData } from '../../import/import-types';
+import { RequestType } from '../health-checks-types';
+
+export const appResourcesWithHealthChecksEnabled: AppResources = {
```

Comment:
Again, all this duplicate code here. I am guessing this is the same object that you have in the file above? You could have easily imported the one from the other file and used it instead.